thirdparty/hairer/radau5_superlu_complex.h

```diff
@@ -0,0 +1,15 @@
+#ifndef _RADAU_SUPERLU_COMPLEX_H_
+#define _RADAU_SUPERLU_COMPLEX_H_
+
+struct SuperLU_aux_z;
+typedef struct SuperLU_aux_z SuperLU_aux_z;
+
+#define min(a,b) ((a) <= (b) ? (a) : (b))
```

Comment:
Do we use min?

Reply:
Actually yes:

// min(nnz_jac + n, n*n) is upper bound on storage requirement of linear system
    slu_aux->data_sys = doublecomplexMalloc(min(slu_aux->nnz_jac + slu_aux->n, n*n));